### ISSUE TITLE
.pot file

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/Easy-Digital-Downloads#1384]
Hi. I'm translating EDD to Finnish and boy there are lot's of strings!

I was wondering if .pot file is up to date? When I click Update (Update Catalogue) in Poedit I get new strings but when I choose Catalogue >> Update from POT file I don't get new strings.

One reason why I ask this is that Poedit might not be the best tool for generating .pot file. Naturally there are lot's of tools for this and WP have their own.

http://i18n.svn.wordpress.org/tools/trunk/

I've read about this [from Tom McFarlin post](http://tommcfarlin.com/internationalize-wordpress-plugins/) and been playing around with it lately. 

I made a [follow up post](https://foxnet-themes.fi/2013/07/07/creating-pot-file-for-your-theme-or-plugin/) for that using W7.

In short benefits of using i18n tools are:
- It pick's up translators notes like this `/* translators: */`.
  - This can be useful for example when translating string `Total earnings for period shown:`. It's hard to pick up that there is empty string after words. Using `ex_()` functions is another solution.
- It picks up header info from plugins and themes. That way there is no need for extra file `languages/strings.php`.
- It pick's up custom page template names automatically but that is more like for themes.

Poedit can't do those things. At least not that I know of.

I think @fxbenard might want to comment on this one.
